backend/mock/service/k8smock/k8smock.go

```diff
@@ -117,6 +117,9 @@ func (*svc) UpdateStatefulSet(ctx context.Context, clientset, cluster, namespace
 	return nil
 }
 
+func (*svc) DeleteStatefulSet(ctx context.Context, clientset, cluster, namespace, name string) error {
+}
```

Comment:
```suggestion
     return nil
}
```

Reply:
Just pushed up my other local changes. This should already be addressed